<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>functionConfigDlg</class>
 <widget class="BaseDlg" name="functionConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">functionConfigDlg</string>
  </property>
  <property name="t" stdset="0">
   <string notr="true">tt</string>
  </property>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>927</x>
     <y>744</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>797</x>
     <y>744</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QWidget" name="sub_widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>990</width>
     <height>616</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>600</height>
    </size>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">all</string>
   </property>
   <widget class="QTableView" name="tableView_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>390</width>
      <height>615</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>44</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>44</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>44</number>
    </attribute>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>146</y>
      <width>118</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="top_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>置顶</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="up_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>向上</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="down_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>向下</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bottom_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>置底</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView_right">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>1</y>
      <width>390</width>
      <height>615</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>53</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="wait_select_num_label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>2</y>
      <width>291</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string>备选（？）</string>
    </property>
   </widget>
   <widget class="QLabel" name="select_num_label">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>2</y>
      <width>211</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string>已选（？）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空全部</string>
    </property>
    <property name="bks" stdset="0">
     <string>transparent</string>
    </property>
    <property name="borders" stdset="0">
     <string>none</string>
    </property>
    <property name="colors" stdset="0">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_v">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1</width>
      <height>617</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_v_2">
    <property name="geometry">
     <rect>
      <x>779</x>
      <y>0</y>
      <width>1</width>
      <height>615</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_v_3">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>0</y>
      <width>1</width>
      <height>617</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
   <widget class="QFrame" name="line_h_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>615</y>
      <width>990</width>
      <height>1</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lines" stdset="0">
     <string notr="true">solid</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>functionConfigDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1011</x>
     <y>764</y>
    </hint>
    <hint type="destinationlabel">
     <x>1110</x>
     <y>713</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
